src/Data/Transport.elm

```diff
@@ -33,7 +38,9 @@ type alias Distances =
 
 type alias Transport =
     { road : Length
+    , roadCooled : Length
     , sea : Length
+    , seaCooled : Length
     , air : Length
     , impacts : Impacts
     }
```

Comment:
Is it possible/wanted to have mixed cooled and non-cooled transports? If yes, scratch the rest of this comment, but if not, how about having a simple boolean doing this distinction? eg. 

```elm
type alias Transport =
    { cooled : Bool
    , road : Length
    , sea : Length
    , air : Length
    , impacts : Impacts
    }
```

Reply:
Yes it's indeed possible to mix cooled and non-cooled transports (eg: Mango is first transported with a non-cooled truck to the logistics center, and then cooled all the way to the transformation site in France)